pwwka.gemspec

```diff
@@ -25,5 +25,5 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency("mono_logger")
   s.add_development_dependency("rake")
   s.add_development_dependency("rspec")
-  s.add_development_dependency("resque")
+  s.add_dependency("resque")
```

Comment:
This will require everyone to install resque, but I don't think we want to do that, right?